onnxruntime/core/platform/windows/logging/etw_sink.h

```diff
@@ -71,6 +71,8 @@ class EtwRegistrationManager {
 
   void RegisterInternalCallback(const EtwInternalCallback& callback);
 
+  void UnregisterInternalCallback(const EtwInternalCallback& callback);
```

Comment:
nit: looks like there's duplicated functionality for handling the callbacks (Register/Invoke/Unregister) that could maybe be split out into a helper class that WindowsTelemetry and EtwSink use. not needed for this PR but something to keep in mind next time the setup is being modified.

Reply:
yea, next time